core/commands/SubprojectDeleteCommand.php

```diff
@@ -0,0 +1,114 @@
+<?php
+# MantisBT - A PHP based bugtracking system
+
+# MantisBT is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# MantisBT is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with MantisBT.  If not, see <http://www.gnu.org/licenses/>.
+
+require_api( 'constant_inc.php' );
+require_api( 'config_api.php' );
+require_api( 'helper_api.php' );
+require_api( 'project_api.php' );
+require_api( 'project_hierarchy_api.php' );
+
+use Mantis\Exceptions\ClientException;
+
+/**
+ * A command that deletes a project subproject.
```

Comment:
We should clarify that we are deleting the sub-project relationship rather than the delete the sub-project itself (as in delete project).  In the Postman APIs that I have added, I used "add/update/delete subproject relationship".

Reply:
Are you asking to simply clarify this in the comments, or reflect this in the commands too, by renaming them to something like `SubprojectRelationshipAdd/Update/DeleteCommand`?

Reply:
I was talking about the comments, but now that you mention it, reflecting in the command names is even better.